airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/AbstractSourceConnectorTest.java

```diff
@@ -221,8 +221,6 @@ private AirbyteSource prepareAirbyteSource(ResourceRequirements resourceRequirem
     if (memoryLimit.isBlank() || memoryLimit.isEmpty()) {
       memoryLimit = WorkerUtils.DEFAULT_RESOURCE_REQUIREMENTS.getMemoryLimit();
     }
-    LOGGER.error("cpu limit -->> {}", cpuLimit);
-    LOGGER.error("memory limit -->> {}", memoryLimit);
```

Comment:
I think it's still worth logging these arguments, maybe as `info`.

Reply:
Returned the logs and changed the logging level to _info_ and log text